bin/merge

```diff
@@ -90,3 +97,59 @@ else
     echo "error: ${repo}#${pull_request} has already been merged"
     exit 1
 fi
+
+}
+
+usage() {
+
+cat << EOF
+Usage: $0 [-h] [-r "REPO-NAME PR-NUMBER"]
```

Comment:
The latest push:

The following two runs of `merge` will print out its usage.
`./merge`
`./merge -h`

Once you need to merge you can run it as usual:

`./merge quasar-analytics/quasar 2030`